### ISSUE TITLE
Is there a way to add headers to UltimateRecyclerView as listview.addHeader(...)? 

### ISSUE THREAD

**Issue by caprice** [cymcsg/UltimateRecyclerView#222]

**Comment by jjhesk** [comment]
yes. this is will very good suggestion and I realised that it is very important to adapt such fashion in the core object in order to make it more extendable. no, we are still working on how to resolved the existing issues from the previous problems. For this reason, we have opened up the new official branch just to make this happening. @caprice

**Comment by jjhesk** [comment]
please now read #276

**Comment by caprice** [comment]
Thx! I will try the new feature.
